### DISCUSSION TITLE
Custom header when using ObsidianNew command

### DISCUSSION THREAD

**Discussion by peeeyow** [epwalsh/obsidian.nvim#483] [Q&A]
Hello!
Previously, before [commit](https://github.com/epwalsh/obsidian.nvim/blob/9cfd03c47a14ebeea6e5d52a240e5dfe64fd2847/lua/obsidian/note.lua#L526) was applied the header of the files I'm creating only uses the title without its path. Currently, the path is now also included in the header. Is there a way to customize this one?

E.g.
Before: `:ObsidianNew permanent/New Title` -> `# New Title`

After: `:ObsidianNew permanent/New Title` -> `# permanent/New Title`

**Comment by epwalsh** [comment]
Hey @peeeyow, this was a regression. Just fixed in f82d96a.
